### ISSUE TITLE
pacs-connector master branch jdk 1.8 and dcm4chee 2.18.3

### ISSUE THREAD

**Issue by OleSta** [nroduit/weasis-pacs-connector#41]
Looks like dcm4chee 2.18.3 can run with jdk 1.8 with small changed to deploy/jmx-console.war/inspectMBean.jsp
(change replace("\"","&quot;") to replaceAl("\"","&quot;")).
Can I use weasis pacs-connector master branch if I use jdk 1.8 or is servlet container support still a problem with dcm4chee-2.18.3 and jdk 1.8?

**Comment by nroduit** [comment]
No, there is a specific implementation for dcm4chee 2.18 in the branch 6.x
